### PR TITLE
fix: nxlog use time field as signal if missing

### LINKED ISSUE #230: Support NXlog without value dataset
This appears to be a valid case, used for top-dead-center chopper timestamps. Currently there is a warning triggered. Probably all there is to do is avoid the warning in this case, by using the `time` dataset as the `signal`?

Should this return the `time` as a `DataArray`, or simply as a `Variable` (as it is now)?

### PR DESCRIPTION
**Pull request by jokasimr**
Fixes #230

### CODE REVIEW

File: tests/nxlog_test.py [hunk 1]
```diff
@@ -52,16 +52,14 @@ def test_nxobject_log(h5root):
     assert sc.identical(log[...], da)
 
 
-@pytest.mark.filterwarnings("ignore:Failed to load /entry/log:UserWarning")
 def test_nxlog_with_missing_value_triggers_fallback(nxroot):
```
Comment (SimonHeybrock) [inline tests/nxlog_test.py:55]: Test name needs update

File: src/scippnexus/nxdata.py [hunk 2]
```diff
@@ -489,6 +489,13 @@ def __init__(self, attrs: dict[str, Any], children: dict[str, Field | Group]):
             fallback_signal_name='value',
         )
 
+    def _init_signal(self, *args, children, **kwargs):
+        super()._init_signal(*args, children=children, **kwargs)
+        if self._signal is None:
+            if 'time' in children:
+                self._signal_name = 'time'
+                self._signal = children['time']
```
Comment (SimonHeybrock) [inline src/scippnexus/nxdata.py:497]: Could this simply use
```python
fallback_signal_name='value' if 'value' in children else 'time'`
```
?

Comment (jokasimr) [reply src/scippnexus/nxdata.py:497]: That's better :+1:

Comment (jokasimr) [reply src/scippnexus/nxdata.py:497]: But, there's no argument to `_init_signal` called `fallback_signal_name`

Comment (SimonHeybrock) [reply src/scippnexus/nxdata.py:497]: My point was to change `__init__` and remove `_init_signal`.

Comment (jokasimr) [reply src/scippnexus/nxdata.py:497]: Aha I see

**Review by SimonHeybrock — APPROVED** [review]

**Pull request merged**